sphinx.conf

```diff
@@ -262,7 +262,7 @@ searchd
  read_timeout  = 5
  max_children  = 30
  pid_file   = /var/run/sphinx/searchd.pid
- max_matches   = 2000
+ max_matches   = 1000
```

Comment:
Hmm. Then Nouman must've made it. Does it ring a bell? It raises flags for me b/c it deals with limits on results. But if it only has to do with front-end stuff, then I'm not too worried about it.